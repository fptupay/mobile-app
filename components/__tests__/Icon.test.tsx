import * as React from 'react'
import renderer from 'react-test-renderer'
import CustomIcon from '../Icon'

it('should render correctly', () => {
  const tree = renderer
    .create(<CustomIcon name="ChevronLeft" color="gray" size={30} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
