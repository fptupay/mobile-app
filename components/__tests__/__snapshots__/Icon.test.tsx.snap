// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="30"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={30}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="gray"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth={2}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 30,
        "width": 30,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={30}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
        "stroke",
        "strokeWidth",
        "strokeLinecap",
        "strokeLinejoin",
      ]
    }
    stroke={
      {
        "payload": 4286611584,
        "type": 0,
      }
    }
    strokeLinecap={1}
    strokeLinejoin={1}
    strokeWidth="2"
  >
    <RNSVGPath
      d="m15 18-6-6 6-6"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
